src/app/components/home/home.component.html

```diff
@@ -39,7 +39,7 @@ <h1>All projects in one place.</h1>
 <!--share image and button-->
 <div class="row share">
   <div class="col-lg-6 col-md-12 image-wrapper">
-    <img src="assets/images/illustrations/undraw_mobile_development_8gyo.svg" alt="" loading="lazy">
+    <img src="assets/images/illustrations/undraw_mobile_development_8gyo.svg" alt="Person working on the floor next to a phone" loading="lazy">
```

Comment:
Is it needed to give alt-text to decorative images?

Reply:
No, I'll elaborate on this further in the PR's discussion